### PR TITLE
change opeing move color

### PR DESCRIPTION
**Pull request by MrBrain295**
as gimps wanted

### CODE REVIEW

File: src/script.ts [hunk 1]
```diff
@@ -655,3 +655,10 @@ function scrollToCurrentGame(): void {
     lastGame.scrollIntoView();
   }
 }
+function changeOpeningMoveColor() {
+  const color = "#c5c5c5";
```
Comment (MrBrain295) [inline src/script.ts:659]: the color is a placeholder and i will change that later

**Comment by MrBrain295** [comment]
if possible i would like to make this optional (or based on user input) but idk how to do that.

**Comment by truekendor** [comment]
@MrBrain295 Thanks for your contribution

Book/opening moves color selection is a great idea
I'll make this optional later along with other customization features

File: src/script.ts (hunk 1 above)
Comment (truekendor) [reply src/script.ts:659]: lgtm
Later will make it optional

**Review by truekendor — APPROVED** [review]

**Comment by MrBrain295** [comment]
also the default CSS `orange` was the old opening move color

**Pull request merged**